.github/workflows/messagingsample.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     runs-on: macos-latest
```

Comment:
My preference would be adding another job in the existing messaging.yml file

Reply:
You can only have one name and I wonder if I can just merge jobs and folders under one name. let me try that.

Reply:
Each job can be separately named